Resources/views/CRUD/list_email.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field%}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- elseif field_description.options.as_string is defined and field_description.options.as_string -%}
+        {{ value }}
+    {%- else -%}
+        {%- set url = '' -%}
+        {%- if field_description.options.subject is defined and field_description.options.subject is not empty -%}
+            {%- set subject = field_description.options.subject -%}
+            {%- set url = 'subject=' ~ subject -%}
```

Comment:
Code between `list_email.html.twig` and `show_email.html.twig` is mostly the same.

It would be great to avoid duplication with code refactoring.

Two solutions:

1. A common template like `_value_email.html.twig` (not sure for the name) which can be included
2. A twig extension. Something like `value | mailto_link(subject, body)`.

For options `2`, we have to decide where to put the extension. Admin? Core? Maybe an external library?

BTW, maybe a package exists providing useful twig extension like this.